api/controllers/pr/findPrs/index.js

```diff
@@ -76,7 +124,37 @@ const findPrs = (github, username) => {
       return Promise.all(checkMergeStatus).then(mergeStatus =>
         _.zipWith(prs, mergeStatus, (pr, merged) => _.assign(pr, { merged }))
       );
-    });
+    })
+    .then(prs => {
+      const checkApproval = _.map(prs, pr => {
+        const repoDetails = pr.repo_name.split('/');
+        const pullDetails = {
+          owner: repoDetails[0],
+          repo: repoDetails[1],
+          number: pr.number
+        };
+
+        return github.pullRequests
+          .getReviews(pullDetails)
+          .then(logCallsRemaining)
+          .then(res => res.data.some(review => review.state === 'APPROVED'));
+      });
+
+      return Promise.all(checkApproval).then(approvalStatus =>
+        _.zipWith(prs, approvalStatus, (pr, approved) =>
+          _.assign(pr, { approved })
+        )
+      );
+    })
+    .then(prs =>
+      prs.filter(
+        pr =>
+          !pr.repo_must_have_topic ||
+          pr.merged ||
+          pr.approved ||
+          pr.has_hacktoberfest_label
+      )
```

Comment:
Not tested this but just looking, shouldn't it be...

```javascript
pr.has_hacktoberfest_label || 
(	
	pr.repo_must_have_topic && 
	(pr.merged || pr.approved)
)
```

> The PR is labelled as hacktoberfest-accepted by a maintainer OR
  Submitted in a repo with the hacktoberfest topic AND (
    The PR is merged OR
    The PR has been approved
  )

Reply:
Oh didn't realize there was a 3rd update to the rules, I've updated the code to match these latest rules.